dmt/main/views.py

```diff
@@ -616,6 +616,28 @@ class NodeDeleteView(LoggedInMixin, DeleteView):
 class MilestoneDetailView(LoggedInMixin, DetailView):
     model = Milestone
 
+    def post(self, request, pk):
+        items = request.POST.getlist('_selected_action')
+        assign_to = request.POST.get('assigned_to')
+
+        assignee = get_object_or_404(UserProfile, username=assign_to)
+
+        item_names = []
+        for pk in items:
+            item = get_object_or_404(Item, iid=pk)
+            item.reassign(request.user.userprofile, assignee, '')
+            item_names.append(
+                '<a href="{}">{}</a>'.format(
+                    item.get_absolute_url(), item.title))
+
+        msg = 'Assigned the following items to <strong>{}</strong>: {}'.format(
+            assignee.get_fullname(),
+            ', '.join(item_names))
+
+        messages.success(request, mark_safe(msg))
+
+        return super(MilestoneDetailView, self).get(request, pk)
```

Comment:
After a POST, it should redirect to the milestone page rather than just displaying output. That prevents back button and reload issues. It's using the `messages` framework, so the messages should still carry through.